kong/plugins/datadog/schema.lua

```diff
@@ -80,6 +80,10 @@ return {
           { service_name_tag = { type = "string", default = "name" }, },
           { status_tag = { type = "string", default = "status" }, },
           { consumer_tag = { type = "string", default = "consumer" }, },
+          { common_tags = {
+              type = "array",
+              elements = { type = "string", match = "^.*[^:]$" },
+          }, },
```

Comment:
This is a breaking change for version compatibility with older data planes; updates will need to be made to the [compatibility layer ](https://github.com/Kong/kong/blob/master/kong/clustering/compat/removed_fields.lua)to handle these changes ahead of time of before a release.